de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/browser/BrowserSXEBridge.java

```diff
@@ -0,0 +1,95 @@
+package de.fu_berlin.inf.dpp.whiteboard.ui.browser;
+
+import org.apache.log4j.Logger;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import de.fu_berlin.inf.dpp.whiteboard.net.WhiteboardManager;
+import de.fu_berlin.inf.dpp.whiteboard.sxe.ISXEMessageHandler.MessageListener;
+import de.fu_berlin.inf.dpp.whiteboard.sxe.SXEController;
+import de.fu_berlin.inf.dpp.whiteboard.sxe.net.SXEMessage;
+import de.fu_berlin.inf.dpp.whiteboard.sxe.records.ElementRecord;
+
+/**
+ * BrowserSXEBridge is the java interface between the SXEController which
+ * contains all model data and the browser which contains the whiteboard and
+ * manipulates the data.
+ */
+public class BrowserSXEBridge {
+
+    private static final Logger LOG = Logger.getLogger(BrowserSXEBridge.class);
+
+    private static final Gson GSON = new Gson();
+
+    private static Gson GSONonlyExposed = new GsonBuilder()
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "GSONonlyExposed" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3008))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=250&issues=AWY_InU7nzfF1D4JrUtZ&open=AWY_InU7nzfF1D4JrUtZ)

[ISSUE_KEY]: AWY_InU7nzfF1D4JrUtZ